### ISSUE TITLE
destroy() removes existing events

### ISSUE THREAD

**Issue by csarven** [yabwe/medium-editor#1023]
https://github.com/yabwe/medium-editor/pull/643 introduces a bug where existing events on the element (and its children) get lost.

**Comment by nmielnik** [comment]
You've seen the events on the actual element get removed?  It makes a lot of sense why child elements would lose their events, but it'd be surprising that setting `.innerHTML` would affect the events on the element itself.

**Comment by csarven** [comment]
Yes, the children lose their events. I don't think the fix to remove the spellchecker should come at the cost of removing events that were added by ME.

**Comment by nmielnik** [comment]
Yeah, this has been around for a decent amount of time though, we should probably look for a way to fix the spell check issue without losing all the event handlers.

Jeremy did mention http://stackoverflow.com/a/9325528/569101 as an alternative so setting `.innerHTML` as part of #639 but it's not clear why that hand't worked...

**Comment by csarven** [comment]
From SO, I've tried:
```
element.spellcheck = false;
element.focus();
element.blur();
```
works in Ubuntu/Firefox 45.0.1. Doesn't work in Chromium Version 49.0.2623.108 Ubuntu 15.10 (64-bit)
